Autor.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
```suggestion
<?php

```
Agregar una línea después de <?php